pulsar-broker/src/test/java/org/apache/pulsar/broker/service/persistent/PersistentStickyKeyDispatcherMultipleConsumersTest.java

```diff
@@ -107,6 +109,9 @@ public void setup() throws Exception {
         HierarchyTopicPolicies topicPolicies = new HierarchyTopicPolicies();
         topicPolicies.getMaxConsumersPerSubscription().updateBrokerValue(0);
 
+        OrderedExecutor orderedExecutor = OrderedExecutor.newBuilder().build();
```

Comment:
shouldn't this use the field?
```suggestion
        orderedExecutor = OrderedExecutor.newBuilder().build();
```

Reply:
good catch !